src/ImageIO/CGImageAnimation.cs

```diff
@@ -0,0 +1,131 @@
+//
+// ImageIO - CGImageAnimation.cs
+//
+// Authors:
+//	Whitney Schmidt  <whschm@microsoft.com>
+//
+// Copyright 2020, Microsoft Corp.
+//
+
+using System;
+using System.Runtime.InteropServices;
+using CoreFoundation;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace ImageIO {
+
+    public partial class CGImageAnimation {
```

Comment:
It looks like this file doesn't follow our coding standards, because it seems to be using spaces to indent instead of tabs.